resources/factions/australia_2009.json

```diff
@@ -0,0 +1,70 @@
+{
+  "country": "Australia",
+  "name": "Australia 2009",
+  "authors": "Khopa, SpaceEnthusiast",
+  "description": "<p>The Australian army in 2009.</p><p>Some units might not be accurate, but were picked to represent at best this army.</p>",
+  "aircrafts": [
+    "AH-1W SuperCobra",
+    "C-130J-30 Super Hercules",
+    "F/A-18C Hornet (Lot 20)",
+   "EA-18G Growler",
+   "F/A-18F Super Hornet",
+    "SH-60B Seahawk",
+    "UH-1H Iroquois"
+  ],
+  "awacs": [
+    "E-3A"
+  ],
+  "tankers": [
+    "KC-130",
+    "KC-135 Stratotanker"
+  ],
+  "frontline_units": [
+    "FV510 Warrior",
+    "LAV-25",
+    "Leopard 1A3",
+    "M113",
+    "M1A2 Abrams"
+  ],
+  "artillery_units": [],
+  "logistics_units": [
+    "Truck M818 6x6"
+  ],
+  "infantry_units": [
+    "Infantry M249",
+    "Infantry M4",
+    "MANPADS Stinger"
+  ],
+  "preset_groups": [
+    "Hawk",
+    "Rapier"
+  ],
+  "naval_units": [
+    "DDG Arleigh Burke IIa",
+    "LHA-1 Tarawa",
+    "L02 Canberra",
+    "HMAS HOBART DDG39"
+  ],
+  "missiles": [],
+  "air_defense_units": [
+    "SAM Hawk SR (AN/MPQ-50)",
+    "HEMTT C-RAM Phalanx"
+  ],
+  "requirements": {
```

Comment:
We don't actually support this field, but it shouldn't cause any issues afaik so no need to remove this...